### DISCUSSION TITLE
Certificate is Invalid in Chrome

### DISCUSSION THREAD

**Discussion by bomshteyn** [laravel/valet#1301] [General]
Over the last few weeks keep getting 

![Screen Shot 2022-11-14 at 11 53 47 AM](https://user-images.githubusercontent.com/4259699/201719122-b6c80dd8-6f6c-4d99-a930-b63524285b99.png)

All the domains used to work ok earlier.

Here are some other users having similar issues: https://twitter.com/jerclarke/status/1589658132480827392

Any suggestions?

**Comment by modernben** [comment]
This helped me

https://stackoverflow.com/questions/72355724/domain-not-secured-with-valet-3-laravel

1. rm ~/.config/valet/CA/*
2. valet install
